README.md

```diff
@@ -146,14 +146,20 @@ Native modules that are designed to work with [N-API](https://nodejs.org/api/n-a
 "binary": {
   "napi_versions": [2,3]
 }
-``` 
+```
 
-In the absense of a need to compile against a specific N-API version, the value `3` is a good choice as this is the N-API version that was supported when N-API left experimental status. 
+In the absence of a need to compile against a specific N-API version, the value `3` is a good choice as this is the N-API version that was supported when N-API left experimental status. 
 
 Modules that are built against a specific N-API version will continue to operate indefinitely, even as later versions of N-API are introduced. 
 
 ### Defining the `NAPI_VERSION` Value
 
+The N-API header files supplied with Node use the w`NAPI_VERSION` preprocessor value supplied by the user to configure each build to the specific N-API version for which the native add-on is being built. In addition, the module's C/C++ code can use this value to conditionally compile code based on the N-API version it is being compiled against.
+
+`prebuild` supports two backend build systems: [`node-gyp`](https://github.com/nodejs/node-gyp) and [`cmake-js`](https://github.com/cmake-js/cmake-js). The `NAPI_VERSION` value is configured differently for each build system. 
```

Comment:
```suggestion
`prebuild` supports two build backends: [`node-gyp`](https://github.com/nodejs/node-gyp) and [`cmake-js`](https://github.com/cmake-js/cmake-js). The `NAPI_VERSION` value is configured differently for each backend. 
```

For consistency; the existing [help section](https://github.com/prebuild/prebuild#help) calls them "build backends".